content/bots/twitterbots/accidental575_es.md

```diff
@@ -0,0 +1,15 @@
+/*
+Title: @accidental575 (es)
+Description: Soy un robot / que encuentra haiku en Twitter / hecho por accidente
+Author: hectorpalmatellez
+Language: es
+Date: 30 de Octubre, 2015
+Tags: twitter,twitterbot,active,retweet,haiku,poetry,eusonic
+
+Nav: hidden
+Robots: index,follow
+*/
+
+[![](/content/bots/twitterbots/images/accidental575.png)](https://twitter.com/accidental575)
+
+[@accidental575](https://twitter.com/accidental575) es un bot de Twitter creado por [Cameron Spencer](https://twitter.com/eusonic) que "encuentra [haikus](https://es.wikipedia.org/wiki/Haiku) en Twitter, hecho por accidente".
```

Comment:
This is a tricky one, because in Spanish none of the texts used counts as a haiku :sob: 
It is just a literal translation.